frontend/src/components/ViewTrips/trip.js

```diff
@@ -8,6 +8,19 @@ import DeleteTripButton from './delete-trip-button.js';
 import ViewActivitiesButton from './view-activities-button.js';
 import * as DB from '../../constants/database.js';
 
+/**
+ * A trip object containing the data stored in a trip document in Firestore.
+ * @typedef {Object} TripData
+ * @property {string} title The trips's title.
+ * @property {string} description A description of the trip.
+ * @property {string} destination The general destination of the trip.
+ * @property {firebase.firestore.Timestamp} start_date Start date Firestore
+ *     timestamp object.
+ * @property {firebase.firestore.Timestamp} end_date End date Firestore
+ *     timestamp object
```

Comment:
Is timestamp an object type that should have backticks?

Reply:
In the way I wrote it here maybe? So I changed it to be `Firestore.Timestamp` because that does make more sense to reference it that way.